pom.xml

```diff
@@ -98,6 +98,7 @@
   <modules>
     <module>sdk</module>
     <module>examples</module>
+    <module>contrib/kafka</module>
```

Comment:
This we do not want to do, I think, as it would probably break users if we don't push the contrib to Maven.

Instead, I think we actually just need to modify travis to ensure that the contrib builds.